blog/en/blog/2022/09/21/apache-apisix-v3-preview.md

```diff
@@ -0,0 +1,201 @@
+---
+title: "Apache APISIX 3.0: 10 Highlights of the Open Source API Gateway"
+authors:
+  - name: Ming Wen
+    title: Author
+    url: https://github.com/moonming
+    image_url: https://avatars.githubusercontent.com/u/26448043
```

Comment:
Was anyone else involved in edit and/or translation? I suggest give them credit as well.

Reply:
Hey @yzeng25, actually this post is improved mainly by Keer and me 😅 (Grammarly of course)